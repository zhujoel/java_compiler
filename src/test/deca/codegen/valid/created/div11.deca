{
    int a = 10.0;
    int b = 5.0;
    print(a / b);
    a = a / b;
    print(a);
}
