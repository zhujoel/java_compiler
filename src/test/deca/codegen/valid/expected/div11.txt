2.00000e0
